doc/STRUCTURE.md

```diff
@@ -0,0 +1,65 @@
+
+# File Structure of the rav1e 0.4.0
+
+
+## High-level directory structure
+
+![Image](https://i.imgur.com/UiFieT5.png)
```

Comment:
I'd prefer to have this image hosted in the git repo. I think that's possible with markdown in Github.